### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by MrZyr0**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -6,14 +6,11 @@ on:
...
   workflow_dispatch:
 
-
 jobs:
-
   build:
     name: Deploy on VPS
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```
Comment (MaxenceMottard) [inline .github/workflows/deploy.yml:16]: Why remove that. It's used to pull files from repo

File: docker-compose.yml [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
+## THIS IS FOR PRODUCTION ONLY ##
+
+version: '3.9'
 
 services:
   db:
-    image: mariadb
-    restart: always
+    image: mariadb:10.7.1
+    restart: on-failure
```
Comment (MaxenceMottard) [inline docker-compose.yml:8]: Please use ´always´ to restart on VPS reboot

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (MrZyr0) [reply .github/workflows/deploy.yml:16]: It's a CMD + Z fail sorry

File: docker-compose.yml (hunk 2 above)
Comment (MrZyr0) [reply docker-compose.yml:8]: okay 👍

**Review by MaxenceMottard — APPROVED** [review]

**Pull request merged**